package/src/utilities/addDevToolbarPluginStubs/preact.ts

```diff
@@ -0,0 +1,22 @@
+import Component from "@@COMPONENT_SRC@@";
+import { h, render } from "preact";
+
+export default {
+	id: "@@ID@@",
+	name: "@@NAME@@",
+
+	// biome-ignore lint/style/noUnusedTemplateLiteral: Using backticks here because most likely copy+pasted svg paths will use double quotes
```

Comment:
could backticks be part of the svg? if yes we need a proper escaping logic here

Reply:
Hmmmm I don't think so? They will never be from a functionality wise because it'll need to be valid HTML so that will only use double quotes. But I guess they could have backticks in there somewhere maybe?

Do you have ideas for how to escape it if they do? Just replace them with single quotes.